src/features/editing/bpmnModelerBuilder.ts

```diff
@@ -12,8 +12,12 @@ export class BpmnModelerBuilder {
     return contents.replace(/(\r\n|\n|\r)/gm, ' ');
   }
 
+  private replaceSingleQuotes(contents: string): string {
+    return contents.replace(/'/gm, "&#39;");
```

Comment:
What is the issue here? Is it VSCode or us messing up?

Reply:
Like I'm wondering if this is valid XML; If so we shall fix this in moddle not this library.

Reply:
Works in Camunda Desktop Modeler: 

![image](https://user-images.githubusercontent.com/58601/194840457-5d023403-a3ce-41d0-85ce-e62968d89c74.png)

Reply:
It is a valid XML, therefore I think that shouldn't be fixed in moddle. I tried a few roundtrips between the extension, our bpmn-io demo, and the Desktop Modeler.

VS Code messes the single quotes up. As we currently go the simple way and provide [all webview content as a single string](https://github.com/bpmn-io/vs-code-bpmn-io/blob/master/src/features/editing/bpmnModelerBuilder.ts#L53), it simply breaks once the content string has another `'` in between.